docs/bigquery-usage.rst

```diff
@@ -455,11 +448,12 @@ located on Google Cloud Storage.  First, create the job locally:
    >>> from gcloud import bigquery
    >>> client = bigquery.Client()
    >>> table = dataset.table(name='person_ages')
-   >>> job = table.export_to_storage(bucket_name='bucket-name',
-   ...                               object_name_glob='export-prefix*.csv',
-   ...                               destination_format='CSV',
-   ...                               print_header=1,
-   ...                               write_disposition='truncate')
+   >>> job = client.extract_table_to_storage(
+   ...          'extract-person-ages', table,
```

Comment:
OCD much? :)

For docstrings, I don't really care:  what would you want it to be?